docs/examples/secrets/README.md

```diff
@@ -0,0 +1,38 @@
+# Secrets examples
+
+To run these examples you need to [create a Tenant](../README.md#tenant) and [set the environment variable](../README.md#pipelinerun) `TENANT_NAMESPACE` accordingly.
+
+## Basic examples
+
+You can find some basic examples in [secretExamples.yaml](secretExamples.yaml).
+
+## Missing secrets
+
+If a secret is missing the pipelien run will fail with an content error.
+
+```bash
+spr=$(kubectl -n "$TENANT_NAMESPACE" create -f pipelinerun_missing_secret.yaml -oname)
+kubectl  -n "$TENANT_NAMESPACE" get $spr -owide
+```
+
+```
+NAME                  STARTED   FINISHED   STATUS     RESULT          MESSAGE
+missingsecret-rsww9   12s                  finished   error_content   ERROR: preparing failed ...
+```
+
+## Secret renaming
+
+It is possible to [rename secrets](../secrets/Secrets.md). This can be tested with the pipeline run in the file [pipelinerun_secret_rename.yaml](pipelinerun_secret_rename.yaml). As preparation you need to create the secret with the rename annotation. You can find the renamed secret in the run namespace as listed below.
```

Comment:
```suggestion
It is possible to [rename secrets](../secrets/Secrets.md).
This can be tested with the pipeline run in file [`pipelinerun_secret_rename.yaml`](pipelinerun_secret_rename.yaml).
As a preparation you need to create the secret with the rename annotation.
You can find the renamed secret in the run namespace as listed below.
```